bin/pd-nagios

```diff
@@ -38,10 +38,8 @@ class NagiosEvent:
     SERVICE_STATE = "SERVICESTATE"
 
     REQUIRED_FIELDS = {
-        "host": {HOST_NAME, HOST_STATE},
-        "service": {HOST_NAME,
-                    SERVICE_DESCRIPTION,
-                    SERVICE_STATE}
+        "host": set([HOST_NAME, HOST_STATE]),
+        "service": set([HOST_NAME, SERVICE_DESCRIPTION, SERVICE_STATE])
```

Comment:
🤔 looks like the set literal should be available [as of python 2.7][1]. But if this works, it works.

  [1]: https://docs.python.org/3/whatsnew/2.7.html#other-language-changes

Reply:
Good catch -- and yeah, still somewhat of a mystery, but can investigate more following this revert.